flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -34,7 +34,7 @@
 /**
  * Context object with optional arguments for a Flink Scan.
  */
-class ScanContext implements Serializable {
+public class ScanContext implements Serializable {
```

Comment:
Why do we need to make so much more of this ScanContext public?

Reply:
because now this class is accessed by classes in sub package (like `reader/RowDataIteratorBulkFormat.java`)